rmf_demos_tasks/setup.py

```diff
@@ -26,11 +26,13 @@
         'console_scripts': [
           'request_loop = rmf_demos_tasks.request_loop:main',
           'request_lift = rmf_demos_tasks.request_lift:main',
-          'dispatch_loop = rmf_demos_tasks.dispatch_loop:main',
```

Comment:
Let's not remove `dispatch_loop` yet as users may still be using the script to submit loop requests. We could update the script to print a deprecation warning and inform users to use `dispatch_patrol` instead. We can consider removing the script for `humble` release where "breaking" things would be acceptable.